docs/dev/docker-hub.md

```diff
@@ -123,54 +123,54 @@ INFO  2016-08-04 06:57:40,840 [shard 0] storage_service - Thrift server listenin
 
 ### Configuring data volume for storage
 
-You can use Docker volumes to improve performance of Scylla.
+You can use Docker volumes to improve the performance of ScyllaDB.
 
-Create a Scylla data directory ``/var/lib/scylla`` on the host, which is used by Scylla container to store all data:
+Create a Scylla data directory ``/var/lib/scylla`` on the host, which is used by the ScyllaDB container to store all the data:
 
 ```console
 $ sudo mkdir -p /var/lib/scylla/data /var/lib/scylla/commitlog /var/lib/scylla/hints /var/lib/scylla/view_hints
 ```
 
-Launch Scylla using Docker's ``--volume`` command line option to mount the created host directory as a data volume in the container and disable Scylla's developer mode to run I/O tuning before starting up the Scylla node.
+Launch ScyllaDB using Docker's ``--volume`` command line option to mount the created host directory as a data volume in the container and disable ScyllaDB's developer mode to run I/O tuning before starting up the ScyllaDB node.
 
 ```console
 $ docker run --name some-scylla --volume /var/lib/scylla:/var/lib/scylla -d scylladb/scylla --developer-mode=0
 ```
 
 ### Configuring resource limits
 
-The Scylla docker image defaults to running on overprovisioned mode and won't apply any CPU pinning optimizations, which it normally does in non-containerized environments.
+The ScyllaDB docker image defaults to running on overprovisioned mode and won't apply any CPU pinning optimizations, which it normally does in non-containerized environments.
 For better performance, it is recommended to configure resource limits for your Docker container using the `--smp`, `--memory`, and `--cpuset` command line options, as well as 
 disabling the overprovisioned flag as documented in the section "Command-line options".
 
-### Restart Scylla
+### Restart ScyllaDB
 
-The Docker image uses supervisord to manage Scylla processes. You can restart Scylla in a Docker container using
+The Docker image uses supervisord to manage ScyllaDB processes. You can restart ScyllaDB in a Docker container using
```

Comment:
```suggestion
The Docker image uses a supervisor to manage ScyllaDB processes. You can restart ScyllaDB in a Docker container using:
```

Reply:
supervisord is the name of a system, I'll make it bold so it's more clear